utilities/src/main/java/io/syndesis/qe/bdd/validation/DbValidationSteps.java

```diff
@@ -186,6 +186,12 @@ public void checkValuesExistInTable(String query) {
             "Query has no output.");
     }
 
+    @Then("^.*checks that query \"([^\"]*)\" has output \"([^\"]*)\"$")
```

Comment:
can't you use `@Then("^.*checks? that query \"([^\"]*)\" has \"(\\w+)\" output$")`?

Reply:
ok